ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/DefendantRepresentativeMapper.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.hmcts.cmc.ccd.mapper;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.ccd.domain.CCDDefendant;
+import uk.gov.hmcts.cmc.domain.models.legalrep.Representative;
+
+import static org.apache.commons.lang3.StringUtils.isBlank;
+
```

Comment:
why is this class part of this pull request . Isn't this supposed to be part of defendant mapping story

Reply:
claimant can provide defendant representative details if have for legal claims